test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -159,6 +179,14 @@ class V2DeparturesControllerSpec
   ): Request[A] =
     FakeRequest(method = method, uri = uri, headers = headers, body = body)
 
+  def fakeAttachDepartures[A](
+    method: String,
+    headers: FakeHeaders,
+    uri: String = routing.routes.DeparturesRouter.attachMessage("123").url,
```

Comment:
if this is always going to be for attachDepartures, you don't need the url to be a parameter

Reply:
That is required as FakeRequst without url does not comply